assets/kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -145,6 +145,7 @@ spec:
         - kube-apiserver
         args:
         - "--openshift-config=/etc/kubernetes/apiserver-config/config.yaml"
+        - "--profiling=false"
```

Comment:
Adding checks for ROKS 4.9 only

Reply:
I think this should be added to the Kubernetes API server config: https://github.com/openshift/ibm-roks-toolkit/blob/master/assets/kube-apiserver/config.yaml